docs/roman/associations/skycell_asn.rst

```diff
@@ -7,6 +7,28 @@ Create an association using either the command line tool
 ``skycell_asn`` or through the Python API using
 :func:`romancal.associations.skycellasn.skycell_asn`
 
+Running this command requires that you have the patch to the
+file containing the division of the sky into patches. This is done
+by setting the environment variable PATCH_TABLE_PATH.
+As an example:
+::
+
+   export PATCH_TABLE_PATH=<location of my patch table>
+
+.. Note::
+
+   The patch table will be available in the CRDS system soon.
+
+
+.. Note::
+
+   **For STScI Users Only:**
+    Users at STScI may access the required
+    data files from the Central Storage network. Set the following
+    environment variables in your ``bash`` shell. (You will probably
+    want to add this to your bash setup file.) ::
+
+      export PATCH_TABLE_PATH="/grp/roman/scsb/tesselation/patches.asdf"
```

Comment:
Same comments as above - we need to better define how to use the file from CRDS.